src/utils/forms.ts

```diff
@@ -366,11 +367,28 @@ const buildEnumSimpleType = (name: string, enumeration: string[]): string => {
  * @param xsdTemplate - XSD template including E-form metadata and some basic types (EnumerationType, PrilohaType)
  * @returns XSD schema
  */
-export const loadAndBuildXsd = (jsonSchema: JsonSchema, xsdTemplate: string | undefined = defaultXsdTemplate): string => {
+export const loadAndBuildXsd = (
+  jsonSchema: JsonSchema,
+  xsdTemplate: string | undefined = defaultXsdTemplate
+): string => {
   const $ = cheerio.load(xsdTemplate, { xmlMode: true, decodeEntities: false })
 
   const mergedJsonSchema = mergeJsonSchema(jsonSchema)
   const properties = getAllPossibleJsonSchemaProperties(mergedJsonSchema)
   buildXsd($(`xs\\:schema`), 'E-formBodyType', mergedJsonSchema.required, properties, [])
   return $.html()
 }
+
+/**
+ * Generate mock data from JSON schema.
+ *
+ *
+ * @param jsonSchema - JSON schema
+ * @returns mock data
+ */
+export const fakeData = (jsonSchema: JsonSchema) => {
+  JSONSchemaFaker.format('data-url', () => JSONSchemaFaker.random.randexp('^[\\w,\\s-]+\\.[A-Za-z]{3}$'))
+  JSONSchemaFaker.format('ciselnik', () => JSONSchemaFaker.random.randexp('[a-zA-Z]+'))
+
+  return JSONSchemaFaker.generate(jsonSchema)
```

Comment:
can this be seeded in a way where it always generates the same output for the same schema ? would be nice if we're gitting the data.json

Reply:
Now we can use example (or examples) property in JSON schema to return a single value from example (or random value from examples).